cosmos-tidoop-api/README.md

```diff
@@ -0,0 +1,209 @@
+#<a name="top"></a>Tidoop REST API
+
+* [What is cosmos-tidoop-api](#section1)
+* [Installation](#section2)
+    * [Prerequisites](#section2.1)
+    * [API installation](#section2.2)
+    * [MySQL database installation](#section2.3)
+    * [Unit tests](#section2.4)
+* [Configuration](#section3)
+* [Running](#section4)
+* [Usage](#section5)
+* [Administration](#section6)
+    * [Logging traces](#section6.1)
+    * [Database](#section6.2)
+* [Contact](#secton7)
+
+##<a name="section1"></a>What is cosmos-tidoop-api
+cosmos-tidoop-api exposes a RESTful API for running MapReduce jobs in a shared Hadoop environment.
+
+Why emphasize in <i>a shared Hadoop environment</i>? Because shared Hadoops require special management of the data and the analysis processes being run (storage and computation). There are tools like [Oozie](https://oozie.apache.org/) in charge of running MapReduce jobs as well through an API, but they do not take into account the access to the run jobs, their status, results, etc must be controlled. In other words, using Oozie any user may kill a job by knowing its ID; using cosmos-tidoop-api only the owner of the job will be able to.
+
+The key point is to relate all the MapReduce operations (run, kill, retrieve status, etc) to the user space in HDFS. This way, simple but effective authorization policies can be stablished per user space (in the most basic approach, allowing only a user to access it own user space). This can be easily combined with authentication mechanisms such as [OAuth2](http://oauth.net/2/).
+
+[Top](#top)
+
+##<a name="section2"></a>Installation
+This is a software written in JavaScript, specifically suited for [Node.js](https://nodejs.org) (<i>JavaScript on the server side</i>). JavaScript is an interpreted programming language thus it is not necessary to compile it nor build any package; having the source code downloaded somewhere in your machine is enough.
+
+###<a name="section2.1"></a>Prerequisites
+This REST API has no sense if tidoop-mr-lib is not installed. And tidoop-mr-lib has only sense in a [Hadoop](http://hadoop.apache.org/) cluster, thus both the library and Hadoop are required.
+
+As said, cosmos-tidoop-api is a Node.js application, therefore install it from the official [download](https://nodejs.org/download/). An advanced alternative is to install [Node Version Manager](https://github.com/creationix/nvm) (nvm) by creationix/Tim Caswell, whcih will allow you to have several versions of Node.js and switch among them.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/a46e1731daedd1c5d9c75e1ca08e90be37aff51a